src/test/java/org/zalando/nakadi/service/subscription/StreamingContextTest.java

```diff
@@ -29,8 +30,8 @@ public void onException(final Exception ex) {
             }
 
             @Override
-            public void streamData(final byte[] data) throws IOException {
-
+            public OutputStream getOutputStream() {
+                return null;
```

Comment:
Based on this, is it right to say there's no test coverage on the new outputstream use?